src/els_code_lens.erl

```diff
@@ -61,7 +61,11 @@ default_lenses() ->
 
 -spec enabled_lenses() -> [lens_id()].
 enabled_lenses() ->
-  els_config:get(code_lenses).
+  Config = els_config:get(lenses),
+  Default = default_lenses(),
+  Enabled = maps:get("enabled", Config, []),
+  Disabled = maps:get("disabled", Config, []),
+  lists:usort((Default ++ valid(Enabled)) -- valid(Disabled)).
```

Comment:
I wonder if we shouldn't extract this as a standard function? We are using it for the diagnostics logic too.

Reply:
I thought about it, but maybe that would make just things harder to read. As long as Elvis does not complain about duplicated code I guess we are fine.